ui/app/services.js

```diff
@@ -208,9 +208,6 @@
             case 404:
               Notifications.error('Requested resource not found');
               break;
-            case 409:
```

Comment:
The correct error messages display now. With this 409 switch case in place the UI always returns the "build still running" error message even though 409 is used to notify the user of conflicts that are not necessarily a result of a running build, such as trying to create a configuration with the same name as an existing one.

This didn't require any back end changes because the UI was ignoring the true error message and returning its own, the REST API was doing the correct thing.

All the endpoints that return a 409 look good to me, the UI should be able to parse them.